samples/python/tutorial_code/dnn/dnn_conversion/common/utils.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import errno
+import os
+import random
+
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+import torch
+
+SEED_VAL = 42
+DNN_LIB = "DNN"
+# common path for model savings
+MODEL_PATH_ROOT = os.path.join(DNN_LIB.lower() + "_conversion", "{}/models")
+
+
+def make_dir(dir_to_create):
```

Comment:
you can replace the function with `os.path.makedirs(dir_to_create, exist_ok=True)`